docs/content/preview/reference/configuration/yb-tserver.md

```diff
@@ -848,6 +848,40 @@ In addition, as this setting does not propagate to PostgreSQL, it is recommended
 --ysql_pg_conf_csv="ssl_min_protocol_version=TLSv1.2"
 ```
 
+## Packed row flags (Beta)
```

Comment:
to confirm, we're explicitly NOT adding these to yb-master, because of the quirk that master needs YCQL?